src/scss/_sofie-dialog.scss

```diff
@@ -0,0 +1,36 @@
+$actions-size-height: 80px;
```

Comment:
Why is this a global CSS file instead of being styling encapsulated by the DialogComponent?

Reply:
Same question for having a global CSS file for Forms?